lib/ModuleReason.js

```diff
@@ -4,9 +4,45 @@
 */
 "use strict";
 
-module.exports = class ModuleReason {
+const util = require("util");
+
+class ModuleReason {
 	constructor(module, dependency) {
 		this.module = module;
 		this.dependency = dependency;
+		this._chunks = null;
```

Comment:
I guess it's because when creating the module reasons the chunk graph is not created and `module._chunks` is null/empty.